docs/contributor/arch/029-batch-processor-migration-to-exporterhelper.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Migration from Batch Processor to Exporter Helper Batching
+status: Proposed
+date: 2025-12-09
+---
+
+# 29: Migration from Batch Processor to Exporter Helper Batching
+
+## Context
+
+The OpenTelemetry Collector's `batchprocessor` has known limitations and is planned for
+deprecation ([opentelemetry-collector#13582](https://github.com/open-telemetry/opentelemetry-collector/issues/13582)).
+The most significant limitation is its inability to propagate backpressure to clients due to its asynchronous behavior,
+forcing operators to choose between supporting batching or backpressure, but not both simultaneously.
+
+A new batching solution has been integrated into exporters through the `exporterhelper` package, tracked
+under [issue #8122](https://github.com/open-telemetry/opentelemetry-collector/issues/8122). This solution aims to
+provide both batching capabilities and proper backpressure propagation.
+
+### Current Situation
+
+The testing setup consists of:
+
+- One primary OTel Collector that receives telemetry from clients
+- One healthy backend OTel Collector receiving forwarded telemetry
+- A fanout configuration where telemetry is sent to multiple backends simultaneously
+
+Backpressure is simulated by configuring the primary OTel Collector to send to the healthy OTel Collector and an
+additional non-existing endpoint (e.g., unreachable endpoint).
```

Comment:
We don't use latin abbreviations, so "e.g." would be replace with "for example"

```suggestion
additional non-existing endpoint (for example, unreachable endpoint).
```

But then what do you mean with this? An unreachable endpoint is an example for a non-existing endpoint? Isn't it the other way round, "non-existing" is one way among others (network error, wrong configuration...) how an endpoint could be unreachable?

Or did you mean something like this?

```suggestion
additional non-existing endpoint (representing an unreachable endpoint).
```